### PR TITLE
Run tests on JDK 17

### PR DESCRIPTION
**Pull request by tgodzik**

### CODE REVIEW

**Comment by tgodzik** [comment]
Blocked on https://github.com/scalameta/metals/pull/3003

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -12,22 +12,21 @@ jobs:
...
             shard: 1
-          - os: windows-latest
-            java: 8
+          - os: ubuntu-latest
+            java: 'adopt@1.8'
             shard: 2
-          - os: macOS-latest
-            java: 8
+          - os: ubuntu-latest
+            java: 'zulu@1.17.0-0'
```
Comment (sideeffffect) [inline .github/workflows/ci.yml:25]: It's probably better to just use `17` here instead of the `zulu...`

**Comment by tgodzik** [comment]
Finally the JDK 17 tests seems to be passing :tada:

**Review by ckipp01 — APPROVED** [review]
This is great that this is now working. We'll have to remember to update the requirements in the docs when we release to reflect this:  https://scalameta.org/metals/docs/editors/vscode#requirements

**Comment by tgodzik** [comment]
> This is great that this is now working. We'll have to remember to update the requirements in the docs when we release to reflect this: https://scalameta.org/metals/docs/editors/vscode#requirements

Updated the docs!

**Pull request merged**